balancer-js/examples/contracts/veBAL-proxy.ts

```diff
@@ -1,23 +1,22 @@
-import { BalancerSDK } from '../../src/modules/sdk.module';
-import { Network } from '../../src';
-import dotenv from 'dotenv';
+/**
+ * This example shows how to the adjusted veBAL balance from the active boost delegation contract
```

Comment:
I feel there's something missing from this sentence, but I couldn't make sense of what 😅

Reply:
perhaps ..how to get the adjusted veBal balance.. ?